src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php

```diff
@@ -87,6 +105,21 @@ private function __construct(array $thresholds = [], $regex = '', $verboseOutput
             }
             $this->verboseOutput[$group] = (bool) $status;
         }
+
+        if ($generateBaseline && empty($baselineFile)) {
+            throw new \InvalidArgumentException('You cannot use the "generateBaseline" configuration option without providing a "baselineFile" configuration option.');
+        }
+        $this->generateBaseline = $generateBaseline;
+        $this->baselineFile = $baselineFile;
+        if (!empty($this->baselineFile)) {
+            if (is_file($this->baselineFile)) {
+                $this->baselineDeprecations = (array) json_decode(file_get_contents($this->baselineFile));
+            } elseif ($this->generateBaseline) {
+                $this->writeBaseline();
+            } else {
+                throw new \InvalidArgumentException(sprintf('The baselineFile "%s" does not exist.', $this->baselineFile));
```

Comment:
we could simply ignore this case silently IMHO

Reply:
I think it is helpful to fail when people set a baseline file and it can't be read or doesn't exist.